app/src/main/java/com/khaithu/a3ctech_android_final_project/view/VideoPlayerActivity.kt

```diff
@@ -0,0 +1,102 @@
+package com.khaithu.a3ctech_android_final_project.view
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import com.google.android.youtube.player.*
+import com.khaithu.a3ctech_android_final_project.BuildConfig
+import com.khaithu.a3ctech_android_final_project.R
+import com.khaithu.a3ctech_android_final_project.helper.Constant
+import com.khaithu.a3ctech_android_final_project.model.ResultVideo
+import com.khaithu.a3ctech_android_final_project.model.enum.VideoEnum
+import com.khaithu.a3ctech_android_final_project.presenter.GetVideoPresenter
+import com.khaithu.a3ctech_android_final_project.view.interfaceView.IVideoPlayerView
+import kotlinx.android.synthetic.main.activity_video_player.*
+
+class VideoPlayerActivity : YouTubeBaseActivity(), IVideoPlayerView {
+
+    private lateinit var onInitializedListenerPlayer: YouTubePlayer.OnInitializedListener
+    private lateinit var onInitializedListenerThumbnail: YouTubeThumbnailView.OnInitializedListener
+    private lateinit var result: ResultVideo
+    private val mPresenter: GetVideoPresenter = GetVideoPresenter(this)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_video_player)
+        val id: Int = intent.getIntExtra(Constant.intentPlayVideo, 0)
+        mPresenter.getVideos(id)
+    }
+
+    override fun onVideoReceiver(resultVideos: List<ResultVideo>) {
+
+        resultVideos.forEach { resultVideo: ResultVideo ->
+            if (VideoEnum.TRAILER.value == resultVideo.type) {
```

Comment:
it can be better like: result.isTrailer. => to do this write a bool extension in enum class